docs/conceptual/node/tx-pool.md

```diff
@@ -2,12 +2,67 @@
 title: Transaction Pool
 ---
 
-* What is the Transaction Pool?
 
-* What is priority?
+The transaction pool contains all transactions (signed & unsigned) broadcasted to the network. The pool is responsible for:
 
-* What is dependency?
+1) Checking if transactions are valid or invalid. For example, ensuring the account that issued the transaction has enough funds to pay the associated fees.
 
-* Graph VS Queue
+2) If the transaction is valid, sorting the transactions into 2 transaction queues:
+   - **Ready Queue** -  Contains transactions that can be included in a new pending block and must follow the *exact* order in the ready queue.
 
-* What is the lifecycle of a transaction?
+   - **Future Queue** - Contains transactions that *seem* valid, but the full node has yet to see the remaining transactions it depends on. For example, *Transaction B* is in the future queue and can only be included in the block after *Transaction A* has been included. However, the node is yet to see *Transaction A* in its transaction pool.
+
+Full nodes in the network all have their own local transaction pool and transaction queues. Therefore, full nodes can implement different strategies on how they prioritise importing transactions.
+
+## Transaction Dependencies
+
+Transaction dependencies are defined by the `requires` and `provides` parameters in `ValidTransaction` to build a dependency graph of transactions. This allows the importation of transactions in a correct and linear order.
```

Comment:
`importation` sounds a bit weird to me, I'd rephrase it and add details on the ordering.

```suggestion
Transaction dependencies are defined by the `requires` and `provides` parameters in `ValidTransaction` to build a dependency graph of transactions. Together with `priority` this allows the pool to traverse the graph and produce a valid linear ordering of transactions that should be propagated and included in the next block.
```

Reply:
And link the rust docs https://substrate.dev/rustdocs/master/sp_runtime/transaction_validity/struct.ValidTransaction.html